frontend/module/spa.js

```diff
@@ -5,8 +5,10 @@ export function querySPALinks(rootDir) {
   return [...internalLinks].filter((link) => {
     // use literal href="..." contents, not derived, absolute link.href
     const href = link.getAttribute("href")
-    // relative link or course root
-    return !href.startsWith("/") || href.startsWith(rootDir)
+    // relative link or root directory links to .html files only
+    const isRelativeLink = !href.startsWith("/") || href.startsWith(rootDir)
+    const isHTMLFile = href.split("#").shift().endsWith(".html")
```

Comment:
Since you are changing the behaviour of this function, you should probably update the comment on the function above with this info.

Reply:
ok, thank you. Updated.